### DISCUSSION TITLE
Looking for users (that are migrating off of something) to co-build Langfuse integration with

### DISCUSSION THREAD

**Discussion by skrawcz** [langfuse/langfuse#1159] [Ideas]
### Describe the feature or potential improvement

Hi All,

I know there are people who start off with a simple framework but invariably move-off of it, e.g. LangChain, but want to keep Langfuse integration.

I drive an open source project called [Hamilton](https://github.com/dagworks-inc/hamilton),  a low-abstraction general purpose DAG framework, that users have been migrating to (see links in additional information), however we don't have a Langfuse integration. I'd love to partner with someone to (a) get them onto Hamilton, and (b) add Langfuse integration with it.

Any takers?

### Additional information

https://www.tryhamilton.dev/
https://github.com/dagworks-inc/hamilton
https://hamilton.dagworks.io/en/latest/code-comparisons/langchain/
https://hub.dagworks.io/docs/DAGWorks/conversational_rag/ (simple RAG example)
https://blog.dagworks.io/p/customizing-hamiltons-execution-with (what we'd build the integration on top of)

**Comment by marcklingen** [comment]
Love this, interoperability is a huge thing for Langfuse and Hamilton is the best-in-class abstraction for specific kinds of applications/workflows.

In the end this integration could be interoperable with the rest of Langfuse in a similar way as the Langchain integration. There, users can get a langchain callback handler (native langchain tracing) for a specific subset in their application via `span.get_langchain_handler()`.

Happy to help on this one based on the learnings from the existing Langfuse integrations.
